cfstore/scripts/aggregatebmetadata.py

```diff
@@ -36,27 +15,13 @@ def manage_types(value):
             with open("{{homedir}}"+filename+"_variables_bmetadata.json","w") as writepath:
                 print("")
             
-            #Reads the fields from the file with cf
-            #Alternatively cfdm can be used as such:
-            #   cff = cfdm.read(filename)
-        cff=cf.read("{{fileinput}}",ignore_read_error=True,fmt="NETCDF",aggregate={"relaxed_units":True,"relaxed_identities":True, "exclude":True},recursive=True)  
-        # loop over fields in file (not the same as netcdf variables)
-        variables=[]
-        for v in cff:
-            properties = v.properties()
+    #Reads the fields from the file with cf
+    #Alternatively cfdm can be used as such:
+    #   cff = cfdm.read(filename)
+    cff=cf.read("{{fileinput}}",ignore_read_error=True,fmt="NETCDF",aggregate={"relaxed_units":True,"relaxed_identities":True, "exclude":False,"concatenate":False},recursive=True, chunks=None)  
 
-            if ('standard_name' not in properties and 'long_name' not in properties):
-                properties['long_name'] = v.identity
-            name, long_name = v.get_property('standard_name', None), v.get_property('long_name', None)
-            identity = v.identity()
-            domain = v.domain._one_line_description()
-            size = v.size
-            files = list(v.get_filenames())
-            var = {"identity":identity,"standard_name":name, "long_name":long_name, "cfdm_size":size, "cfdm_domain":domain, "in_files":files}
-            for k,p in properties.items():
-                if k not in ['standard_name','long_name']:
-                    var[k] = manage_types(p) 
-            if var:
-                variables.append(var)
-        with open("{{homedir}}/"+"tempfile.json","w") as writepath:
-            json.dump(variables,writepath)
\ No newline at end of file
+    print("Test",len(cff),[v.data.cfa_get_write() for v in cff])
+    writepath="{{homedir}}/"+"tempfile.cfa"
+    
+    cf.write(cff,cfa={'strict',False},filename=writepath)
```

Comment:
I don't think we need the strict thing now (that we have set `chunks=None` in `cf.read`). best to leave it out until we suspect other wise
```suggestion
    cf.write(cff,cfa=True,filename=writepath)
```